R/error-checks.R

```diff
@@ -414,14 +414,28 @@ validateSameOutputsBetweenSets <- function(simulationSets, logFolder = NULL) {
     pkParametersTable <- getPKParametersInSimulationSet(set)

```

Comment:
Could you update the error message and advise the user they can check the output and pk parameters of simulation sets using the function: `'getPKParametersInSimulationSet'`
Btw, if we decide to not enforce the same outputs, we only need to update this method to throw a warning instead of a stop.

Reply:
Done